#! /bin/sh -e

# DP: don't make device files on install

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p1 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p1 < $0
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

--- isdnutils-3.3.orig/scripts/makedev.sh
+++ isdnutils-3.3/scripts/makedev.sh
@@ -1,4 +1,5 @@
 #!/bin/sh
+exit 0	# not done when building a package for debian
 #
 # $Id: makedev.sh,v 1.5 2000/02/24 13:29:26 paul Exp $
 #
